recipes-kernel/linux/files/patches-6.1/0099-spi-cadence-qspi-Improve-spi-memory-performance.patch

```diff
@@ -0,0 +1,40 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Li Hua Qian <huaqian.li@siemens.com>
+Date: Wed, 9 Apr 2025 13:02:41 +0800
+Subject: [PATCH] spi: cadence-qspi: Improve spi memory performance
+
```

Comment:
Let's mark where this is coming from:

[ commit 3cb2a2f7eebbb0752a834708e720a914e61841a1 upstream ]